gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -306,6 +306,53 @@ gst_tensor_info_get_rank (const GstTensorInfo * info)
   return idx + 1;
 }
 
+/**
+ * @brief Allocate and initialize the extra info in given tensors info.
+ * @param[in,out] info tensors info to be updated.
+*/
+gboolean
+gst_tensors_info_extra_create (GstTensorsInfo * info)
+{
+  GstTensorInfo *new;
+
+  g_return_val_if_fail (info != NULL, FALSE);
+
+  if (info->extra) {
+    nns_logd ("Extra tensors info is allocated already");
+    /* clear existing extra info */
+    gst_tensors_info_extra_free (info);
```

Comment:
Could I know why this function clear existing extra info instead of return FALSE?
It's not suggestion, just a question :)

Reply:
This function does initializing as well as creating. Add new functions for just creating or initializing could be more proper. I'll consider this later, thanks!

Reply:
Thank you for the explanation